web/src/app/components/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthService } from 'src/app/services/auth.service';
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.scss']
+})
+export class DashboardComponent implements OnInit {
+
+  constructor(
+    public authService: AuthService,
+    public router: Router,
+  ) { }
+
+  ngOnInit() {}
+
+  logout() {
+      this.authService.signOutUser()
+        .subscribe(() => this.navigateTologin());
+  }
+
+  navigateTologin() {
+    this.router.navigate(['/login']);
```

Comment:
Indentation is different to above

Reply:
Will fix those indentation issues.

Reply:
Thanks 👍 . I didn't comment on all, please check out all files. Have a look at using the auto format in the IDE on file save 😉